src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.AutoreleasePool.OSX.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: perhaps call it Apple if it targets all apple oses

Reply:
The pattern we have today is to call files for all "OSX-like" systems (macOS, iOS, tvOS) `.OSX.`, so I'm just following the pattern.